lib/types.js

```diff
@@ -62,6 +63,7 @@ module.exports = function (modelTypes) {
   modelTypes.registerType(Array);
   modelTypes.registerType(GeoPoint);
   modelTypes.registerType(Object);
+  modelTypes.registerType(ObjectId);
```

Comment:
It is already registered as an alias in the ObjectId class definition, so it should work properly. I added an e2e test to prove it.